packages/fiori/src/themes/ViewSettingsDialog.css

```diff
@@ -12,13 +12,31 @@
 }
 
 .ui5-vsd-title {
-	margin-bottom: 0.3rem;
+	font-size: 1rem;
+	margin-left: 0;
+}
+
+.ui5-vsd-sub-title {
+	margin-left: 0;
+}
+
+.ui5-vsd-bar-sub-header {
+	padding-left: 0.5rem;
+}
+
+.ui5-vsd-bar-header {
+	box-shadow: none;
+	padding-left: 0.5rem;
```

Comment:
This padding should be on the inner parts of the bar, not on the bar as in the latter case the content of the bar is enlarged and shifted to the right in order to hit the design. The total width of the bar should be the same as the width of the dialog, not larger.

Reply:
done